### PR TITLE
[SDK] Add new feature (iexec Account) into sdk

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

**Review by cedric25 — COMMENTED** [review]
Good job! Some suggestions for you to consider 👍

File: packages/sdk/src/lib/dataProtectorSharing/subscribeToCollection.ts [hunk 1]
```diff
@@ -10,25 +11,33 @@ import {
...
+  onlyValidSubscriptionParams,
+} from './smartContract/preflightChecks.js';
 import { getCollectionDetails } from './smartContract/sharingContract.reads.js';
 
 export const subscribeToCollection = async ({
   iexec = throwIfMissing(),
   sharingContractAddress = throwIfMissing(),
   collectionId,
   duration,
+  price,
```
[the remark this answers is not in the document]
Comment (Le-Caignec) [reply packages/sdk/src/lib/dataProtectorSharing/subscribeToCollection.ts]: 0d6f2524079f0987a3ffb553a523b1b6d7c5cd35

File: packages/sdk/src/lib/dataProtectorSharing/subgraph/getUserAddOnlyAppWhitelistQuery.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ export async function getUserAppWhitelistQuery({
       }
     }
   `;
-  const getUserAppWhitelistQueryResponse: GetUserAppWhitelistGraphQLResponse =
+  const getUserAddOnlyAppWhitelistQueryResponse: GetUserAddOnlyAppWhitelistGraphQLResponse =
     await graphQLClient.request(getCreatorsQuery);
-  return getUserAppWhitelistQueryResponse;
+  return getUserAddOnlyAppWhitelistQueryResponse;
```
[the remark this answers is not in the document]
Comment (Le-Caignec) [reply packages/sdk/src/lib/dataProtectorSharing/subgraph/getUserAddOnlyAppWhitelistQuery.ts]: Commit: 7417c42d9cb14f46817260440d869e8634b80aaf

**Review by cedric25 — APPROVED** [review]

**Comment by cedric25** [comment]
### 📝 Docs

Doc pages to add:
 - getUserAddOnlyAppWhitelist()
 - createAddOnlyAppWhitelist()
 - addAppToAddOnlyAppWhitelist()

Doc pages to update:
 - addToCollection
 - buyProtectedData
 - rentProtectedData
 - setProtectedDataForSale
 - setProtectedDataToRenting
 - setSubscriptionParams

**Pull request merged**